nestkernel/stimulating_backend.cpp

```diff
@@ -0,0 +1,26 @@
+/*
+ *  stimulating_backend.cpp
+ *
+ *  This file is part of NEST.
+ *
+ *  Copyright (C) 2004 The NEST Initiative
+ *
+ *  NEST is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 2 of the License, or
+ *  (at your option) any later version.
+ *
+ *  NEST is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#include "stimulating_backend.h"
+
+const std::vector< Name > nest::StimulatingBackend::NO_DOUBLE_VALUE_NAMES;
+const std::vector< Name > nest::StimulatingBackend::NO_LONG_VALUE_NAMES;
```

Comment:
I don't think this file is actually needed. See my comment on the corresponding header file.

Reply:
This variable and function are deleted because there are not used.